src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj

```diff
@@ -330,4 +336,6 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />
+  <Import Project="$(MSBuildProjectExtensionsPath)$(_TargetAssemblyProjectName)$(MSBuildProjectExtension).nuget.g.targets"
```

Comment:
Hack: https://github.com/dotnet/Nerdbank.GitVersioning/issues/404, since its a vsix with pages

Reply:
Add this comment to the project file itself.

---
In reply to: [521584272](https://github.com/microsoft/sarif-visualstudio-extension/pull/267#discussion_r521584272) [](ancestors = 521584272)